docs/api/conventions.md

```diff
@@ -191,9 +191,9 @@ app/
 
 The file in `app/root.tsx` is your root layout, or "root route" (very sorry for those of you who pronounce those words the same way!). It works just like all other routes:
 
-- You can export a [`loader`](#loader), [`action`](#action), [`meta`](#meta), [`headers`](#headers), or [`links`](#links) function
-- You can export an [`ErrorBoundary`](#errorboundary) or [`CatchBoundary`](#catchboundary)
-- Your default export is the layout component that renders the rest of your app in an [`<Outlet />`](https://reactrouter.com/docs/en/v6/api#outlet)
+- You can export a [`loader`][3], [`action`][4], [`meta`][5], [`headers`][6], or [`links`][7] function
```

Comment:
Is there any way your package could try to use the content of the thing that's being linked for the name rather than a number? So this would be:

```suggestion
- You can export a [`loader`][loader], [`action`][action], [`meta`][meta], [`headers`][headers], or [`links`][links] function
```

It would need to strip characters that don't work in a link (like back-ticks and `<`s I think) and probably truncate at some max-length. And you'd want to handle duplicates with post-fixed numbers.